tests/testthat/test-http.R

```diff
@@ -22,8 +22,8 @@ test_that("authHeaders() picks correct method based on supplied fields", {
   )
 
   # Dummy key created with
-  # openssl::base64_encode(openssl::ed25519_keygen())
-  key <- "MC4CAQAwBQYDK2VwBCIEIDztfEgkp5CX7Jz0NCyrToaRW1L2tfmrWxNDgYyjO9bQ"
+  # openssl::base64_encode(openssl::rsa_keygen(2048L))
+  key <- readLines(test_path("fake-key"))
```

Comment:
PKI doesn't support non-RSA keys. If I understand correctly this is totally fine because we only [generate RSA keys](https://github.com/rstudio/rsconnect/blob/81b18c95c43d33cb395d6bef66b90d6aa5cc2075/R/accounts.R#L150). 

Was the use of ed25519 just because it was shorter? 

I also committed these changes separately to be able to pull that and see if tests past there.

Reply:
I'm guessing so, yes (shorter).

I would also be fine if the `fake-key` were a constant in-code rather than a file living alongside the tests. You could store it in PEM format, which has the start/end markers and (generally) shorter lines.

Reply:
Thanks, I pulled the PEM format into the test. It's not the prettiest, but the sidecar file also isn't either.